scripts/testnet/sovereignBridge/pyScripts/update_toml.py

```diff
@@ -0,0 +1,44 @@
+import os
+import sys
+import re
+
+
+def update_toml(lines, section, identifier, main_chain_address) -> []:
```

Comment:
Please add more explanatory name for this, we actually update the subscribed addresses here, right?